airflow/providers/amazon/aws/operators/glacier.py

```diff
@@ -54,3 +54,55 @@ def __init__(
     def execute(self, context: Context):
         hook = GlacierHook(aws_conn_id=self.aws_conn_id)
         return hook.retrieve_inventory(vault_name=self.vault_name)
+
+
+class GlacierUploadArchiveOperator(BaseOperator):
+    """
+    This operation adds an archive to a vault
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GlacierUploadArchiveOperator`
+
```

Comment:
Small nitpick, personal request/preference:  I see you alphabetized the docstring parameters, I usually put them in the same order they are declared in the init() method sig.  I don't know if there is a "standard" here or not.  Unless someone else chimes in to add more weight to this, consider it a suggestion.

Reply:
Make sense to maintain order at both places